pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -368,10 +380,6 @@ func (ds *dockerService) ListPodSandbox(filter *runtimeapi.PodSandboxFilter) ([]
 	// These PodSandbox will only include PodSandboxID, Name, Namespace.
 	// These PodSandbox will be in PodSandboxState_SANDBOX_NOTREADY state.
 	if filter == nil {
```

Comment:
nit: It seems that we don't need to check the filter here then.

Just small nit, up to you. :)